src/Agent/MsiInstaller/InstallerActions/InstallerActions.csproj

```diff
@@ -1,69 +1,25 @@
-<?xml version="1.0" encoding="utf-8"?>
-<Project ToolsVersion="4.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" Condition="Exists('$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props')" />
+<Project Sdk="Microsoft.NET.Sdk">
+
   <PropertyGroup>
-    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
-    <Platform Condition=" '$(Platform)' == '' ">x86</Platform>
-    <ProductVersion>8.0.30703</ProductVersion>
-    <SchemaVersion>2.0</SchemaVersion>
+    <TargetFramework>net481</TargetFramework>
```

Comment:
This might require net481 to be on the machine running the installer. Should this be net462 instead (the lowest supported .net framework version)?

Reply:
I don't think it will matter since the previous version was targeting net35 and net40 without issue.  I can set the target to our minimum supported one though.